GGO.Singleplayer/Main.cs

```diff
@@ -75,6 +79,16 @@ private void OnTick(object Sender, EventArgs Args)
                         // To end this up, increase the count of peds "rendered"
                         Count++;
                     }
+
+                    // Check for on screen dead Peds to display dead markers, limit to 10 to not clutter the screen.
+                    if (NearbyPed.IsDead && NearbyPed.IsOnScreen && DeadCount < 10)
+                    {
+                        // Draw marker
+                        Draw.DeadMarker(Config, NearbyPed, DeadCount);
+
+                        // Increase the count of dead peds "rendered"
+                        DeadCount++;

```

Comment:
```suggestion
```

Reply:
Limit has been removed, though the counter is stilll needed to draw unique files to the UI screen, as it won't draw the same file multiple times.

Reply:
Use the `Ped.Hash` property to make them unique. 
The hashes are unique per ped and we only need one image so that is ok.

Reply:
I can add this, but I fear it may end up cluttering the %TEMP%/GGO folder. There will be a .png file for every ped hash. With the counter, it reuses the numbered files when drawing.